fabric-api-base/src/test/java/net/fabricmc/fabric/test/base/EventTests.java

```diff
@@ -16,63 +16,59 @@
 
 package net.fabricmc.fabric.test.base;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
 import java.util.function.Consumer;
 import java.util.function.Function;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
 
+import net.minecraft.Bootstrap;
+import net.minecraft.SharedConstants;
 import net.minecraft.util.Identifier;
 
 import net.fabricmc.fabric.api.event.Event;
 import net.fabricmc.fabric.api.event.EventFactory;
 import net.fabricmc.fabric.impl.base.toposort.NodeSorting;
 
 public class EventTests {
-	private static final Logger LOGGER = LoggerFactory.getLogger("fabric-api-base");
-
-	public static void run() {
-		long time1 = System.currentTimeMillis();
-
-		testDefaultPhaseOnly();
-		testMultipleDefaultPhases();
-		testAddedPhases();
-		testCycle();
-		NodeSorting.ENABLE_CYCLE_WARNING = false;
-		testDeterministicOrdering();
-		testTwoCycles();
-		NodeSorting.ENABLE_CYCLE_WARNING = true;
-
-		long time2 = System.currentTimeMillis();
-		LOGGER.info("Event unit tests succeeded in {} milliseconds.", time2 - time1);
+	@BeforeAll
+	static void beforeAll() {
+		SharedConstants.createGameVersion();
```

Comment:
Are these needed for the tests?

Reply:
Probably not, but doesn't really matter. Also note that our testmods are used by devs as templates for other testmods.

Reply:
If its not needed, please remove. At the very least it will make the tests a second or two faster when running on their own.